### PR TITLE
added information if incase of XCL_EMULATION_MODE is not set

### PR DESCRIPTION
**Pull request by vboggara-xilinx**
added information if incase of XCL_EMULATION_MODE is not set

### CODE REVIEW

File: src/runtime_src/xrt/device/hal.cpp [hunk 1]
```diff
@@ -89,7 +89,15 @@ dllpath(const boost::filesystem::path& root, const std::string& libnm)
...
-  static bool val = (std::getenv("XCL_EMULATION_MODE") != nullptr);
+  static int count = 0;
+  static bool val = (std::getenv("XCL_EMULATION_MODE") != nullptr);  
+  if (!val) {
+    if (!count) {
+      std::string initMsg = "INFO: XCL_EMULATION_MODE is not set, make sure that this environment variable is set to valid value for emulation";
+      std::cout << initMsg << std::endl;
+    }
+    count++;
+  }
```
Comment (stsoe) [inline src/runtime_src/xrt/device/hal.cpp]: This logic does not look right. As far as I can tell the message is printed when running in non emulation mode.  The message makes no sense when running in HW.

Comment (vboggara-xilinx) [reply src/runtime_src/xrt/device/hal.cpp]: @stsoe , yes soren i agree with you, is there any other way to handle this error?

Comment (chvamshi-xilinx) [reply src/runtime_src/xrt/device/hal.cpp]: Hi @vboggara-xilinx , There is no way to findout whether you are running on emulation or hw. Main purpose of this Environment variable is to differentiate between both the modes.
This is well documented in all the user guides. We shouldnt give this message while running on board.

**Comment by chvamshi-xilinx** [comment]
Closing this PR as this effects hardware runs too

**Pull request closed without merging**